### ISSUE TITLE
GUI: FED: Switch the GUI to use the new Powersubsystem (PowerSupplies)

### ISSUE THREAD

**Issue by gtmills** [ibm-openbmc/dev#3028]
Is the call for the Power Supplies 

https://github.com/openbmc/webui-vue/blob/602e98aa32f82fd3b0c3d250c7cc1f8da971db24/src/store/modules/Health/PowerSupplyStore.js

Switch to https://${bmc}/redfish/v1/Chassis/chassis/PowerSubsystem/PowerSupplies

Backend: https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/40326

**Comment by dixsie** [comment]
resolved by https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/43366

**Comment by derick-montague** [comment]
@dixsie - It looks like all the FEDs are assigned as reviewers. Can you work with the team to determine who should be assigned and who should be CC'd?

**Comment by gtmills** [comment]
This is downstream. Could we pull in downstream?

**Comment by derick-montague** [comment]
Removed blocked label because dependent story is ready downstream (code review in driver).

**Comment by dixsie** [comment]
Ready for review in GHE

**Comment by dixsie** [comment]
resolved by https://github.ibm.com/openbmc/webui-vue/pull/6

**Comment by gtmills** [comment]
How are we going to track the upstream work for this ? Can we move this to "in driver" after GHE merges?

**Comment by derick-montague** [comment]
> How are we going to track the upstream work for this ? Can we move this to "in driver" after GHE merges?

I think we need to be moving anything that is both up and downstream to the `Code in Driver` pipeline. That is a new process that we need to talk about as a team since we have not had to do that before. I also think we decided to move this downstream after some brief conversation, but I want to avoid the overhead of managing multiple reviews. My feeling is we should abandon any commit in Gerrit that was moved downstream. 

The reason I suggest that is any changes made downstream would have to be ported to gerrit. I would prefer that when the code is approved and merged downstream, we cherry-pick upstream and create a  new review in Gerrit.
